### DISCUSSION TITLE
Vercel Preview - High Score on Lighthouse, low in PageSpeed

### DISCUSSION THREAD

**Discussion by rafalwawrzyk** [vercel/community#512] [Help] [answered]
Hello, i have question about preview in Vercel. I have low results on PageSpeed website checking the branch preview. But on Lightspeed i have great results 20 points higher. Is it possible that preview make results lower and if app will be deployed this results will be higher as in Lighthouse?

**Comment by crowterli** [comment, marked as answer]
Hiya.

They don't function in exactly the same way with regards to scoring even though Page Speed Test implements Lighthouse as part of its testing. There's a good article from the wider internet that discusses this here: https://www.debugbear.com/blog/why-is-my-lighthouse-score-different-from-pagespeed-insights
